distribution/tools/keystore-cli/src/main/java/org/opensearch/cli/keystore/package-info.java

```diff
@@ -9,4 +9,4 @@
 /**
  * Classes implementing a CLI tool for managing secrets in the OpenSearch keystore.
  */
-package org.opensearch.common.settings;
+package org.opensearch.cli.keystore;
```

Comment:
How about making this `org.opensearch.tools.cli.keystore`? The `java-version-checker` and `launcher` modules within this "tools" folder make use of the `org.opensearch.tools` prefix. Seems like a good opportunity to make things consistent and also make the same change to `plugin-cli` and `upgrade-cli`. I don't feel super strongly though.

Reply:
Yes we should do that and keep it consistent.